test/unit/user_test.rb

```diff
@@ -370,7 +370,7 @@ class UserTest < ActiveSupport::TestCase
       end
     end
 
-    context "recommend mfa" do
+    context "mfa recommend or require from downloads" do
```

Comment:
NIT: I think it'd be better to use the same tense as everywhere else. In other words:

```
context "mfa recommended or required from downloads" do
```

Conversely, this could work too:
```
context "recommend or require mfa from downloads" do
```

(PS this "from downloads" wording _really_ discombobulates my understanding for some reason)